scripts/src/release/releaser.py

```diff
@@ -0,0 +1,160 @@
+to the charts and development repositories.
+- Adds the cron job to .github/worklfows/schedule.yml and changes the verifier image used in .github/worklfows/schedule.yml
+  to latest, as required. The charts repo is updated from development repo which necessitates these update.
+- Create a PR against the charts repo containing the workflow updates. This requires manual merge.
+- Directly commits to the development main branch any new charts added to the charts repo since the last update.
+
+
+"""
+import yaml
+import os
+import argparse
+import sys
+from release import release_info
+
+sys.path.append('../')
+from github import gitutils
+
+SCHEDULE_YAML_FILE=".github/workflows/schedule.yml"
+BUILD_YAML_FILE=".github/workflows/build.yml"
+
+SCHEDULE_INSERT = [
+    '  # Daily trigger to check updates',
+    '  schedule:',
+    '    - cron: "0 0 * * *"'
+]
+
+def update_workflow():
+
+    lines=[]
+    with open(SCHEDULE_YAML_FILE,'r') as schedule_file:
+
+        lines = schedule_file.readlines()
+
+        for line in lines:
+            if line.strip() == "on:":
+                insert_location = lines.index(line)+1
+                if SCHEDULE_INSERT[0] not in lines[insert_location].rstrip():
+                    print("[INFO] add cron job to schedule.yaml")
+                    lines.insert(insert_location,f"{SCHEDULE_INSERT[0]}\n")
+                    lines.insert(insert_location+1,f"{SCHEDULE_INSERT[1]}\n")
+                    lines.insert(insert_location+2,f"{SCHEDULE_INSERT[2]}\n")
+                    break
+
+    with open(SCHEDULE_YAML_FILE,'w') as schedule_file:
+        schedule_file.write("".join(lines))
+
+
+    with open(BUILD_YAML_FILE,'r') as build_file:
+
+        lines = build_file.readlines()
+
+        for line in lines:
+            if "VERIFIER_IMAGE:" in line:
+                if "chart-verifier:main" in line:
+                    line_index = lines.index(line)
+                    print(f"replace: {lines[line_index].rstrip()}")
+                    lines[line_index] = lines[line_index].replace('chart-verifier:main','chart-verifier:latest')
+                    print(f"with   : {lines[line_index].rstrip()}")
+
+    with open(BUILD_YAML_FILE,'w') as build_file:
+        build_file.write("".join(lines))
+
+
+def make_required_changes(release_info_dir,origin,destination):
+
+    print(f"Make required changes from {origin} to {destination}")
+
+    repository = "development"
+    if "charts" in origin or "development" in destination:
+        repository = "charts"
+
+    replaces = release_info.get_replaces(repository,release_info_dir)
+
+    for replace in replaces:
+        replace_this=f"{destination}/{replace}"
+        with_this = f"{origin}/{replace}"
+        if os.path.isdir(with_this) or os.path.isdir(replace_this):
+            print(f"Replace directory {replace_this} with {with_this}")
+            if os.path.isdir(replace_this):
+                os.system(f"rm -rf {replace_this}")
```

Comment:
stay native, use shutils or os for directly and file manipulation

Reply:
Try to avoid os.system call unless it is the only option to interact with the system. This will use the language abstraction for os calls etc...

Reply:
Blame stack overflow on this  one lol, will look for some python alternatives. Also I an using os.

Reply:
OK

Reply:
avoided all os.system calls, took a few iterations to get it right but is fully working now.